css/style.css

```diff
@@ -0,0 +1,307 @@
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 10px;
+}
+
+.home-welcome-section2--img-tablet{
+    display: none;
+}
+
+.delivery{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+}
+.delivery > p{
+    padding-bottom: 7px;
+    letter-spacing: 0.03rem;
+}
+
+/* HERO */
+.home-hero{
+    grid-area: hr;
+}
+.home-hero img{
+    width: 100%;
+}
+/* PRODUCTS */
+.products {
+    display: flex;
+    grid-area: pr;
+    justify-content: center; 
+}
+.products-cards h2{
+    text-align: center;
+    padding:1.625rem; 
+    color: var(--wine);
+}
+
+.products img {
+    width: 158px;
+    margin: 8px;
+}
+.products-cards-items{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+/* FOOTER */
+.home-footer {
+    margin-top: 30px;
+    padding-top: 30px;
+    padding-bottom: 100px;
+    text-align: center;
+    grid-area: ft;
+    background-color:var(--cream-light);
+
+}
+
+/* MEDIA QUERIES */
+
+/* Tablet Version */
+@media only screen and (min-width:540px)  {
+    .home {
+    justify-content: center;
+    grid-template-columns:repeat(6, 1fr);
+    grid-template-areas:
+
+    "h    h    h    h    h    h    h   h"
+    "n    n    n    n    n    n    n   n"
+    "hr   hr   hr   w    w    w    w   w"
+    "pr   pr   pr   pr   pr   pr   pr  pr"
+    "ft   ft   ft   ft   ft   ft   ft  ft"; 
+    }
+/* NAV */
+    .home-nav {
+        display: flex;
+        justify-content: center;
+        height: 50px;
+        grid-area: n;
+        background-color:var(--cream-light)
+    }
+
+    .home-nav > img{
+        display: none;
+    }
+
+    .home-nav-links li:first-child {
+        padding-left: 0px;
+}
+
+    .home-nav-links,
+        li,
+        a {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            text-decoration: none;
+            height: 100%
+    }
+
+        .home-nav-links a {
+            color: var(--grey-light);
+    }
+
+
+        .home-nav-links a:hover {
+            color: var(--wine);
+    }
+
+        .home-nav-links li {
+            padding-left: 40px;
+    }
+/* PRODUCTS */
+        .products {
+            display: flex;
+            grid-area: pr;
+            justify-content: center;
+            width: 100%; 
+    }
+
+        .products img {
+            width: 100px;
+            margin: 8px;
+    }
+/* HERO */
+        .home-hero--img{
+            height:100%;
+            object-fit: cover;    
+    }
+
+        .delivery > p {
+            padding-bottom: 9px;
+
+    }
+/* WELCOME */
+        .home-welcome img {
+            height: 44px;
+    }
+        .home-welcome-section2--img-phone{
+        display: none;
+    }
+
+        .home-welcome-section2--img-tablet{
+        display: unset;
+    }
+} 
+
+/* Desktop Version */
+@media only screen and (min-width:768px)  {
+    .home {
+    justify-content: center;
+    grid-template-columns:repeat(6, 1fr);
```

Comment:
it looks like you have defined 6 columns, but your `grid-template-areas` property has 8 columns